WordPress/Classes/ViewRelated/Me/App Settings/Privacy Settings/PrivacySettingsAnalyticsTracker.swift

```diff
@@ -5,34 +5,37 @@ protocol PrivacySettingsAnalyticsTracking {
     func trackPrivacySettingsReportCrashesToggled(enabled: Bool)
 
     func trackPrivacyChoicesBannerSaveButtonTapped(analyticsEnabled: Bool)
-}
-
-final class PrivacySettingsAnalyticsTracker: PrivacySettingsAnalyticsTracking {
 
-    private let tracker: AnalyticsEventTracking.Type
-
-    init(tracker: AnalyticsEventTracking.Type = WPAnalytics.self) {
-        self.tracker = tracker
-    }
+    func track(_ event: PrivacySettingsAnalytics, properties: [String: String])
+}
 
-    // MARK: - API
+extension PrivacySettingsAnalyticsTracking {
 
     func trackPrivacySettingsReportCrashesToggled(enabled: Bool) {
         let props = ["enabled": enabled.stringLiteral]
-        self.track(.privacyChoicesBannerSaveButtonTapped, properties: props)
+        self.track(.privacySettingsReportCrashesToggled, properties: props)
     }
 
     func trackPrivacyChoicesBannerSaveButtonTapped(analyticsEnabled: Bool) {
         let props = ["analytics_enabled": analyticsEnabled.stringLiteral]
         self.track(.privacyChoicesBannerSaveButtonTapped, properties: props)
     }
 
-    func track(_ event: PrivacySettingsAnalytics, properties: Properties = [:]) {
-        let event = AnalyticsEvent(name: event.rawValue, properties: properties)
-        tracker.track(event)
+    func track(_ event: PrivacySettingsAnalytics) {
+        self.track(event, properties: [:])
     }
+}
+
+final class PrivacySettingsAnalyticsTracker: PrivacySettingsAnalyticsTracking {
```

Comment:
[Nit]
The protocol and class names are too similar and it was a bit confusing to me at first. How about:
- Protocol: PrivacySettingsTracker
- Class: PrivacySettingsWPAnalyticsTracker

Reply:
@mokagio As the author of the protocol, what do you think about this suggestion? Personally I dont find the protocol and class names confusing.